ui/src/containers/VolumePage.js

```diff
@@ -92,7 +92,7 @@ const VolumePage = (props) => {
 
   // If data has been retrieved and no volume is selected yet we select the first one
   useEffect(() => {
-    if (volumeListData.length && !currentVolumeName) {
+    if (volumeListData[0]?.volumeAlertsRetrieved && !currentVolumeName) {
```

Comment:
```suggestion
    if (alerts.length && pVCList.length && !currentVolumeName) {
```

The `volumeAlertsRetrieved` is the flag to indicate if `PVC` and `alerts` are ready, add it for each item seem a bit redundant for me.

I would retrieve pVCList from VolumePage.  
```javascript
const pVCList = useSelector((state) => state?.app?.volumes?.pVCList);
```

You may want to add a little comment to quickly explain why we add this check/ what problem does it solve? 

Thanks.

Reply:
Indeed :) thanks.